doc/source/prometheus.rst

```diff
@@ -0,0 +1,31 @@
+====================
+ Prometheus support
+====================
+
+`Prometheus`_ can use Gnocchi to store its data through
+`Remote Write Adapter`_.
+
+Example of Prometheus configuration::
+
+  remote_write:
+  - url: "http://localhost:8041/v1/prometheus/write"
```

Comment:
This does not seem to handle auth and it's likely required.

Reply:
fixed